resources/config/snemo/demonstrator/simulation/geant4_control/2.1/hit_processing/_fix-issue16_validation/run.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
I would move this script into source/flsimulate/tests (merge the current develop branch to pick up the updates from PR #18). AT present, it is too deeply nested and is not run by CTest.